static-assets/ng-views/audit.html

```diff
@@ -219,7 +221,7 @@ <h1>{{ 'admin.audit.AUDIT' | translate }}</h1>
             <tr dir-paginate="log in audit.logs | filter:q | itemsPerPage: audit.logsPerPage"
                 current-page="audit.pagination.current" total-items="audit.totalLogs">
                 <td class="min-cel-width">{{log.operationTimestamp | amTimezone:audit.timeZone | amDateFormat:'MM-DD-YYYY hh:mm:ssa'}}</td>
-                <td>{{log.siteName}}</td>
+                <td>{{ sites.byId[log.siteName] ? sites.byId[log.siteName].name : log.siteName }}</td>
```

Comment:
I think the API now gives you the site name too, so we don't need to keep a separate site lookup, no?